sgrpg.js

```diff
@@ -87,4 +126,11 @@ Hooks.on("renderChatPopout", (app, html, data) => ItemSg.chatListeners(html));
 function getTensionDie() {
     // First, try to get the Tension Die of the currently active scene, if that turns out unset, get the Tension Die of the campaign
     return game.scenes.active?.getFlag("sgrpg", "sceneTensionDie") || game.settings.get("sgrpg", "campaignTension");
+}
+
+/** Returns whether the combat is configured to use Moxie instead of Initiative
+ * @returns {boolean} True for Moxie, False for Initiative
+ */
+function usingMoxieCombat() {
+    return game.settings.get("sgrpg", "useMoxieCombat");
```

Comment:
I believe there was at least one place where this was get using game.settings resulting in code duplicity and possible future issues in case this would change. This getter should be used everywhere